src/models/os.ts

```diff
@@ -362,27 +362,32 @@ const getOsModel = function (
 	};
 
 	const _memoizedGetAllOsVersions = authDependentMemoizer(
-		async (deviceTypes: string[], listedByDefault: boolean | null) => {
+		async (
+			deviceTypes: string[],
+			filterOptions: 'default' | 'include_draft' | 'all',
```

Comment:
`authDependentMemoizer` is atm using `primitive: true` and doesn't expose a normalizer, so I've left it for the next time we need to add an extra property for `getAvailableOsVersions`.